packages/react-reconciler/src/ReactFiberRoot.js

```diff
@@ -151,3 +161,51 @@ export function createFiberRoot(
 
   return root;
 }
+
+export function isRootSuspendedAtTime(
+  root: FiberRoot,
+  expirationTime: ExpirationTime,
+): boolean {
+  const firstSuspendedTime = root.firstSuspendedTime;
+  const lastSuspendedTime = root.lastSuspendedTime;
+  return (
+    firstSuspendedTime !== NoWork &&
+    (firstSuspendedTime >= expirationTime &&
+      lastSuspendedTime <= expirationTime)
```

Comment:
FYI, totally I think it's time to rename expirationTime because it's been talked a few months, but for now, since it's a 'time' could we compare them through the 'time order'? I.e. from small to large (`lastSuspendedTime <= expirationTime` && expirationTime <= firstSuspendedTime)